### PR TITLE
update APIRemovedInNextRelease alerts

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

**Comment by dgrisonnet** [comment]
The e2e tests seem to be failing:
- https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-kube-apiserver-operator/1436/pull-ci-openshift-cluster-kube-apiserver-operator-master-e2e-gcp-operator-single-node/1619035029126516736

It seem that one API is still in use but it is not clear to me which one it is from the test output.

/lgtm cancel

File: bindata/assets/alerts/api-usage.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ spec:
               a successful upgrade to the next EUS cluster version with Kubernetes {{ $labels.removed_release }}.
               Refer to `oc get apirequestcounts {{ $labels.resource }}.{{ $labels.version }}.{{ $labels.group }} -o yaml` to identify the workload.
           expr: >-
-            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1[.]2[67]"})
+            group by (group,version,resource,removed_release) (apiserver_requested_deprecated_apis{removed_release=~"1[.]2[7]"})
```
Comment (soltysh) [inline bindata/assets/alerts/api-usage.yaml:37]: Shouldn't that be `2.[78]` just like we had before?

**Pull request merged**